embroider.py

```diff
@@ -10,10 +10,11 @@
 # Embroidery file format documentation:
 # http://www.achatina.de/sewing/main/TECHNICL.HTM
 
+execfile('inkstitch_activate_venv.py')
+
 import sys
 import traceback
 sys.path.append("/usr/share/inkscape/extensions")
```

Comment:
Just a question, independent of current commit, because I think it can cause problems, but I'm not sure I understood what it does:
1. What is the purpose of this (i.e. what is it used for - for finding the virtual environment? (why was it there before?)
2. Does this interfere with the notion of the user extension directory being the place where users should put their extensions (that in Inkscape is (normally) ~/.config/inkscape/extensions)? (if put in /usr/share/... a user's extensions can potentially be removed on update of Inkscape, which isn't fun...)
3. Does it need to be absolute? I put my self-compiled Inkscape versions into /opt/<branchname>..., and I also use a different user config directory for them. I imagine others might do that, too. There's an environment variable named INKSCAPE_PROFILE_DIR which should give the correct location for the currently running Inkscape instance.

Reply:
This line wasn't there before.  This `execfile` is the crux of this PR.  `inkstitch_activate_venv.py` in turn does an `execfile` on the `bin/activate_this.py` in the virtualenv.

Activating the virtualenv just adds its python modules into the PYTHONPATH -- it doesn't _replace_ the PYTHONPATH.  So the end result here is to allow this extension to access the modules we've installed in the virtualenv.

As to 3, what hard-coding are you speaking of?  All of this gets installed in the user's inkscape extension directory (~/.config/inkscape/extensions for me).  The hardcoding of `inkstitch_activate_venv.py` is fine because Inkscape always sets the current directory to the path of the extension file being run (`embroider.py`, for example).

Reply:
This was about 
`sys.path.append("/usr/share/inkscape/extensions")` - it's unrelated to the current commit.

Reply:
(and I don't understand what it does here, and am not sure it makes sense)

Reply:
Oh.  My mistake, sorry!

Wow, I hadn't even noticed that.  It really oughtn't be necessary, as inkscape should set PYTHONPATH as necessary to give the script access to `inkex.py` et al.  I'll take a look at this soon.

Reply:
Thank you!